Emby.Naming/Audio/AlbumParser.cs

```diff
@@ -23,6 +23,9 @@ public AlbumParser(NamingOptions options)
             _options = options;
         }
 
+        [GeneratedRegex(@"([-\.\(\)]|\s+)")]
+        private static partial Regex CleanRegex();
```

Comment:
This new regex doesn't have the same output

Old: 
`disc- 1` -> `disc 1`
New (note double spaces): 
`disc- 1` -> `disc  1`

Reply:
Good catch